Extract send fees into a page and add scenarios (#1025)
BitcoinDesign/Guide: guide/daily-spending-wallet/sending.md

## Fees

> Fees are a sensitive topic for users and should always be presented transparently with appropriate information and options for users to choose what works for them. The network itself charges fees for payment processing, and a service provider might also charge fees for services they provide in addition (like providing lightning liquidity). For more details, see the [send fees page]({{ '/guide/daily-spending-wallet/sending/send-fees/' | relative_url }}).

Reviewer: ```suggestion
Fees are a sensitive topic for users and should always be presented transparently with appropriate information and options for users to choose what works for them. The network itself charges fees for payment processing, and a service provider might also charge fees for additional services they provide such as lightning liquidity. For more details, see the [send fees page]({{ '/guide/daily-spending-wallet/sending/send-fees/' | relative_url }}).
```

---

> Let's look a closer look at [send feed]({{ '/guide/daily-spending-wallet/sending/send-fees/' | relative_url }}).

Reviewer 2: ```suggestion
Let's look a closer look at [send fees]({{ '/guide/daily-spending-wallet/sending/send-fees/' | relative_url }}).
```
